src/locales/ko/dialogue.ts

```diff
@@ -399,6 +411,18 @@ export const PGMdialogue: DialogueTranslationEntries = {
       1: "하? 내가 졌어?!"
     },
   },
+  "magma_admin": {
+    "encounter": {
+      1: "Hehehe! So you've come all the way here! But you're too late!",
+      2: "You're going to meddle in Team Magma's affairs? You're so cute you're disgusting! I'll put you down kiddy!",
+      3: "I'm going to give you a little taste of pain! Resign yourself to it!"
+    },
+    "victory": {
+      1: "Hehehe... So I lost...",
+      2: "You're disgustingly strong!",
+      3: "Ahahaha! Ouch!"
```

Comment:
```suggestion
  "magma_admin": {
    "encounter": {
      1: "…아하-! ……역시 왔네…그치만 안타깝게 됐어……다 끝났거든-.",
      2: "……남은……내 일은……너를……막는 것.",
      3: "……너랑……인게이지……하고 싶어…아하하하-!!"
    },
    "victory": {
      1: "……룰루리-.",
      2: "……재밌쪄-!",
      3: "…하아하아……으…하아하아…"
```
i added one snippet for src/locales/ko/dialogue.ts
@returntoice work is perfect, but has a problem. 

 "♪" is not supported in current font. (it appeared formal font, not pixel.)
 so i replaced  "♪" to "-!", "-."....etc temporarily with her characteristic.

(↓ not important!  but if you wanna this more, read it)
 "♪"  is used as suffix in Courtney's dialogue in ORAS, 6th pokemon game, japanese&korean version. 
I know her pokerogue's dialogue from RSE, 3rd pokemon. but korean translators(in discord) agreed to use 6rd's because it's more popular for korean user.